src/bin/index.js

```diff
@@ -94,6 +102,120 @@ yargs(hideBin(process.argv))
     async (argv) => await example(argv.name)
   )
   .command(['system', 'sys', 's'], 'Show system info', {}, () => system())
+  .command(
+    ['lightnet <sub-command> [options]'],
+    'Manage the lightweight Mina blockchain for zkApps development and testing purposes.\nYou can find more information about the Docker image in use at\nhttps://hub.docker.com/r/o1labs/mina-local-network',
+    (yargs) => {
+      yargs
+        .command(
+          [
+            'start [mode] [type] [proof-level] [mina-branch] [archive] [sync] [pull]',
+          ],
+          'Start the lightweight Mina blockchain network Docker container.',
+          {
+            mode: {
+              alias: 'm',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetModes,
+              default: 'single-node',
+              description:
+                'Whether to form the network with one node or with multiple network participants.\n"single-node" value will make the network faster.',
+            },
+            type: {
+              alias: 't',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetTypes,
+              default: 'fast',
+              description:
+                'Whether to configure the network to be fast or slower with closer-to-real-world properties.',
+            },
+            'proof-level': {
+              alias: 'p',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetProofLevels,
+              default: 'none',
+              description: '"none" value will make the network faster.',
```

Comment:
@shimkiv  what is the disadvantage of `none` proof-level? `none` is default, but does not test other proofs, right? maybe more description to help user choose the appropriate proof level?

Reply:
It is the protocol configuration. One of the expensive parts of the protocol is creating blockchain snark proofs, so mocking that is important for faster everything. The `proof_level` Mina constant can be used to disable proving (set it to be `check`) or disable block calculation altogether (set it to `none`). `None` is default because for zkApps development with o1js we don't really care about protocol proofs, assuming that it is well tested. Sometimes though one might want to run zkApps against network that has closer to "real world" properties (say, some of the Nightly CI tests). That is when `zk lightnet start --mode multi-node --type real --proof-level full` might be useful.